operate/services/deployment_runner.py

```diff
@@ -97,7 +97,8 @@ class BaseDeploymentRunner(AbstractDeploymentRunner, metaclass=ABCMeta):
 
     def _run_aea(self, *args: str, cwd: Path) -> Any:
         """Run aea command."""
-        return self._run_cmd(args=[self._aea_bin, *args], cwd=cwd)
+        # TODO: Patch for Windows failing hash (add -s). Revert once it's fixed on OpenAutonomy / OpenAEA
+        return self._run_cmd(args=[self._aea_bin, "-s", *args], cwd=cwd)
```

Comment:
This should be reverted when the OpenAutonomy / Open AEA issue is resolved. cc @0xArdi  @Adamantios  @iasonrovis

Reply:
it might be required on other places as well, id suggest you do a search of other uses of the `_aea_bin` and add this flag everywhere.

Reply:
In principle everything should be run through _run_aea, shouldn't it?

Reply:
I also expect that.

Reply:
Addressed. Please re-review